README.md

```diff
@@ -82,11 +76,12 @@ this should execute the `rootlogon.C` of gLike each time you launch root.
 In gLike you find the following directories:
```

Comment:
This is the description of the source directory, maybe we should describe the build directory as well?

Reply:
Added!